### PR TITLE
utils:API to get inventory object path from JSON

### PR DESCRIPTION
**Pull request by PriyangaRamasamy**
Given a D-bus inventory path/redundant EEPROM path/FRU EEPROM path if present in system config JSON, this API returns the D-bus inventory object path.

Test cases covered:
1. Given an inventory dbus object path, get back the same.
2. Given a FRU EEPROM path, get back inventory dbus object path.
3. Given a redundant EEPROM path which is present in JSON, get back inventory dbus object path.
4. Given any path, if not present in JSON, get back the empty string.

### CODE REVIEW

**Review by SunnySrivastava1984 — APPROVED** [review]

**Review by SunnySrivastava1984 — APPROVED** [review]

File: include/utility/json_utility.hpp [hunk 1]
```diff
@@ -550,5 +550,59 @@ inline std::string getFruPathFromJson(const nlohmann::json& i_sysCfgJsonObj,
     }
     return std::string();
 }
+
+/**
+ * @brief Get inventory object path from system config JSON.
+ *
+ * Given either D-bus inventory path/FRU EEPROM path/redundant EEPROM path,
```
Comment (jinuthomas) [inline include/utility/json_utility.hpp:557]: what's the difference between D-bus inventory path and D-bus inventory object path ?

File: include/utility/json_utility.hpp [hunk 2]
```diff
@@ -550,5 +550,59 @@ inline std::string getFruPathFromJson(const nlohmann::json& i_sysCfgJsonObj,
...
+ * this API returns D-bus inventory object path if present in JSON.
+ *
+ * @param[in] i_sysCfgJsonObj - System config JSON object
+ * @param[in] i_vpdPath - Path to where VPD is stored.
+ *
+ * @throw std::runtime_error.
+ *
+ * @return On success return valid path, on failure return empty string.
```
Comment (jinuthomas) [inline include/utility/json_utility.hpp:565]: @return On success returns valid path, on failure returns empty string or an exception.
OR
@return On success a valid path is returned, on failure an empty string is returned or an exception is thrown.

Comment (PriyangaRamasamy) [reply include/utility/json_utility.hpp]: done

File: include/utility/json_utility.hpp (hunk 1 above)
Comment (PriyangaRamasamy) [reply include/utility/json_utility.hpp:558]: no difference :) modified

**Comment by SunnySrivastava1984** [comment]
jenkins -> run tests please

**Review by jinuthomas — APPROVED** [review]

**Pull request merged**